wallet-api/src/Wallet/UTXO/Runtime.hs

```diff
@@ -33,9 +39,9 @@ instance LiftPlc PendingTxOutType
 
 -- | Output of a pending transaction.
 data PendingTxOut = PendingTxOut {
-    pendingTxOutValue   :: Value,
-    pendingTxDataScript :: Maybe Hash,
-    pendingTxOutData    :: PendingTxOutType
+    pendingTxOutValue  :: Value,
+    pendingTxOutHashes :: Maybe (Hash, Hash), -- ^ Hashes of validator script and data script
```

Comment:
Could we type this a bit better so it's not so easy to mix up the two hashes? Likewise below.